### DISCUSSION TITLE
Links between blocks of postman are disappearing

### DISCUSSION THREAD

**Discussion by prasanna31** [postmanlabs/postman-flows#305] [Defect]
<img width="566" alt="Screenshot 2022-04-27 at 6 48 05 PM" src="https://user-images.githubusercontent.com/54216599/165527306-caf4b514-c3f5-4aad-97b9-4d562b6f2fee.png">

This has happened to me in near past too.. I had a complete flow which ran completely fine and when I opened it on the next day those links between the blocks disappeared. I had no choice other than doing it again. 

Can some one suggest me how can I get this flow back to working?

**Comment by prasanna31** [comment]
Can someone take a look at this.

**Reply by samay-v** [reply]
Hey @prasanna31 
Could you download a `Snapshot` of the flow and share with us here? 
<img width="456" alt="image" src="https://user-images.githubusercontent.com/28357106/166240291-1b34522a-d0b7-43d6-9556-e98a8975ff0e.png">
